entry/endpoints.cc

```diff
@@ -110,14 +121,26 @@ response Endpoints::generate_credentials(const request& req) {
         cerr << "generate_credentials: " << e->what() << endl;
         return response(401);
     }
-
-    return ret_val;
+    /*
```

Comment:
Please don't comment out code, delete it, make a note if you need it, but we have github for version control